modules/runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/RuntimeMessageHandler.java

```diff
@@ -41,12 +48,51 @@
      * Add new error.
      * <p />
      * Error messages block server startup in strict mode.
+     *
+     * @deprecated since 11.3, use {@link #addMessage(Level, String, String, String, String)} or
+     *             {@link #addMessage(RuntimeMessage)} instead. O
      */
+    @Deprecated
     void addError(String message);
 
     /**
      * @return an unmodifiable {@link List} of error messages
      */
     List<String> getErrors();
 
+    void addMessage(RuntimeMessage message);
+
+    /**
+     * Adds the following message.
+     * <p>
+     * The source is a free string identifier allowing to distinguish between messages (and to clear them selectively if
```

Comment:
It's not a `free string identifier` :)

Reply:
Forgot to update the javadoc after latest changes indeed